assets/styles/components/_entry.scss

```diff
@@ -291,6 +291,13 @@
     align-items: center;
     column-gap: 0.25rem;
 
+    .user-badge {
+      border: var(--kbin-section-border);
+      padding: 0.25rem 0.5rem;
+      margin-left: .25rem;
+      border-radius: var(--kbin-rounded-edges-radius);
+    }
```

Comment:
instead of copying this section we should make it global shouldn't we?

Reply:
I've never seen a good place for global css, if you have any suggestions let me know. CSS is divided into components, mixins, and variables. components only have css that effect their single component. layout, similar, only have things that effect the class they're named after (e.g. `_options.scss` starts with `.options {` so only effects things under that

I could instead change this to a mixin so it isn't repeated, but that will still increase the file size by the same amount this copy paste does